pyanaconda/modules/users/users.py

```diff
@@ -100,6 +100,9 @@ def process_kickstart(self, data):
             group_data.name = group_ksdata.name
             if group_ksdata.gid is not None:
                 group_data.gid = group_ksdata.gid
+                group_data.gid_mode = ""
```

Comment:
This looks unclean to me. You are using the _mode field like an enum, thus it should have a defined value (`ID_MODE_USE_{DEFAULT,VALUE}`), and `_VALUE` seems appropriate here as you just set one? Having a third nameless mode is confusing IMHO. Same thing several times below.

Reply:
Ha, good catch, that empty string should definitely not be there.

Reply:
This code is outdated.